### ISSUE TITLE
not passing device integrity Lineage OS

### ISSUE THREAD

**Issue by jj15warrior** [chiteroman/PlayIntegrityFix#448]
Hello, this morning i lost the device integrity verdict. I tried uninstalling PIF, installing again, using playCURL, while rebooting after every step. I also disabled every module except PIF and it still does not work. ![Screenshot_20241017-124628_Play Integrity API Checker.png](https://github.com/user-attachments/assets/1d01a21f-336a-4240-88f2-00c2b0af250e)

![Screenshot_20241017-124643_Magisk.png](https://github.com/user-attachments/assets/7da4e7df-5505-4b0c-8277-0a655be2ebba)

![Screenshot_20241017-124649_Magisk.png](https://github.com/user-attachments/assets/fd72ceb0-4060-47e2-b897-7dae835ccfb5)

My lineage version is 20-20240430-NIGHTLY-haydn

and device is xiaomi Mi-11i, codename haydn

**Comment by Djtrip83** [comment]
Just to add I experience the same results on the API Integrity Checker and only started today. I'm on a stock Pixel 6a.

**Comment by marcussacana** [comment]
I'm pretty lucky, I just installed this rom today as try to get the PIF

**Comment by ElliesSurviving** [comment]
the current fingerprint is revoked, i think

**Comment by marcussacana** [comment]
The PlayIntegrityFork works in my LineageOS
![Screenshot_20241019-005636_TB Checker](https://github.com/user-attachments/assets/07371a3f-620e-4da6-b7b8-05da6ce975dc)

**Comment by Djtrip83** [comment]
Tested with osmosis fork as well and same result. Both this PlayIntegrityFix and the osmosis version show BASIC integrity but not DEVICE

![Screenshot_20241019-132946.png](https://github.com/user-attachments/assets/5d620652-060f-4916-935b-357a13e127f9)

![Screenshot_20241019-133114.png](https://github.com/user-attachments/assets/93db224b-a11b-4861-8a39-c817b25eb374)

**Comment by marcussacana** [comment]
I'm using those modules as well, maybe one of then is helping pass, my device is a xiaomi.
Sensetive props is a fork by [jun10r4lm31d4](https://github.com/jun10r4lm31d4/sensitive_props/)

![Screenshot_20241019-121221_Settings](https://github.com/user-attachments/assets/8ff15fc4-77d5-40cd-9656-135ae95dfdc6)

**Comment by jj15warrior** [comment]
Kind of solved the issue, i have a module called tricky_store that uses a keybox in /data/adb/tricky_store/keybox.xml and it looks like the provided keybox got banned or somehow detected. I changed it to some random keybox i found online and it started working again (BASIC and DEVICE integrity, not passing STRONG)

**Comment by Djtrip83** [comment]
> Kind of solved the issue, i have a module called tricky_store that uses a keybox in /data/adb/tricky_store/keybox.xml and it looks like the provided keybox got banned or somehow detected. I changed it to some random keybox i found online and it started working again (BASIC and DEVICE integrity, not passing STRONG)

Can confirm for me it was the module "Tricky-Store-v1.1.3-128-820c2b2-release" as well. Removing it, re-installing chiteroman's module and clearing cache of the key Google apps did the trick.

**Comment by Tiego80** [comment]
> > Kind of solved the issue, i have a module called tricky_store that uses a keybox in /data/adb/tricky_store/keybox.xml and it looks like the provided keybox got banned or somehow detected. I changed it to some random keybox i found online and it started working again (BASIC and DEVICE integrity, not passing STRONG)
> 
> Can confirm for me it was the module "Tricky-Store-v1.1.3-128-820c2b2-release" as well. Removing it, re-installing chiteroman's module and clearing cache of the key Google apps did the trick.

Did you reslovet it?

**Comment by Djtrip83** [comment]
Yes - by uninstalling the module affecting this one (see issue #449). However, this issue (#448) is for another device (lineageOS) so check with the others.

**Comment by sniirful** [comment]
It seems like modifying the `/system/etc/font_fallback.xml` in any way also does not let you pass DEVICE integrity.

I'm using [GSP](https://gitlab.com/nongthaihoang/google-sans-prime) with little modifications to make it work on Android 15. Does this mean I cannot load custom fonts anymore?

**Comment by GaPhi** [comment]
Tried on lineageos, ok after removing tricky store module 👍

**Comment by sniirful** [comment]
Honestly it fixed itself with the custom-modified GSP, don't know how, it's weird.
